boot/mynewt/mcuboot_config/syscfg.yml

```diff
@@ -19,8 +19,8 @@
 # Package: boot/mynewt/mcuboot_config
 
 syscfg.defs:
-    BOOTUTIL_VALIDATE_SLOT0:
-        description: 'Validate image at slot 0 on each boot.'
+    BOOTUTIL_VALIDATE_PRIMARY_SLOT:
```

Comment:
When you change the name of this `syscfg` you are basically breaking every Mynewt configuration ever created that wants to have validation on slot0.

Reply:
I agree with this, I think we need to be more careful about changing things that are externally visible, names of features, configuration, etc. I do like the idea of having better terms, but this change is really pervasive, and is making for lots of merge conflicts, etc. I think the feature names should probably be left as SLOT0, just because external entities are already using these names.